src/Arcus.WebApi.Security/Authorization/AzureManagedIdentityAuthorizationFilter.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.Threading.Tasks;
+﻿using System;
+using System.Threading.Tasks;
 using Arcus.WebApi.Security.Authorization.Jwt;
 using GuardNet;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.Filters;
 
 namespace Arcus.WebApi.Security.Authorization
 {
+    [Obsolete("Renamed to " + nameof(JwtTokenAuthorizationFilter))]
     public class AzureManagedIdentityAuthorizationFilter : IAsyncAuthorizationFilter
```

Comment:
Maybe just get rid of everything and do this?
```suggestion
    public class AzureManagedIdentityAuthorizationFilter : JwtTokenAuthorizationFilter  
```

Reply:
Well, yeah, but I changed a bit the `JwtTokenAuthorizationFilter` to make a distinction between a missing header and a header with a wrong value.
So, I didn't want to change the implementation of the "old" filter.

Reply:
Ok, fair enough!